cmd/skaffold/app/cmd/cmd.go

```diff
@@ -205,7 +205,8 @@ func NewSkaffoldCommand(out, errOut io.Writer) *cobra.Command {
 	templates.ActsAsRootCommand(rootCmd, nil, groups...)
 	rootCmd.PersistentFlags().StringVarP(&v, "verbosity", "v", log.DefaultLogLevel.String(), fmt.Sprintf("Log level: one of %v", log.AllLevels))
 	rootCmd.PersistentFlags().IntVar(&defaultColor, "color", int(output.DefaultColorCode), "Specify the default output color in ANSI escape codes")
-	rootCmd.PersistentFlags().BoolVar(&forceColors, "force-colors", false, "Always print color codes (hidden)")
+	rootCmd.PersistentFlags().BoolVar(&forceColor, "force-colors", false, "Always print color codes (hidden)")
```

Comment:
Is there a way I should deprecate this option?

Reply:
Changing the name of the flag is a breaking change, we shouldn't do this.

Reply:
We can keep both `--force-colors` and `--force-color` as flags to avoid any breakage. How does Skaffold deal with informing users about upcoming breaking changes?